### PR TITLE
feat: Create `Toaster` and `Toast` components (#806)

### PR DESCRIPTION
**Pull request by essj**
Issue [#806](https://github.com/binance-chain/ui-project-management/issues/806).

Small screens:
image

Medium+ screens:
image

### CODE REVIEW

File: src/components/Toaster/component.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+export const Component = ({
+  position = 'bottom-center',
+  'data-testid': testId,
+  ...otherProps
+}: Props) => {
+  const { buildTestId } = useBuildTestId({ id: testId });
+
+  const close = useCallback(
+    ({ closeToast }: { closeToast: any }) => (
```
Comment (cchungmin) [inline src/components/Toaster/component.tsx]: Hmm can we have a type for `closeToast`?

File: src/components/Toaster/styled.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    margin-bottom: ${({ theme }) => em(theme.honeycomb.size.small)};
+    padding: 0;
+    background-color: ${({ theme }) => theme.honeycomb.color.bg.normal};
+  }
+  .Toastify__toast-body {
+    display: flex;
+    margin: 0;
+    padding: 0;
+  }
+
```
Comment (cchungmin) [inline src/components/Toaster/styled.tsx]: Is this newline for separating main classes (for me like `toast-body`) and its sub classes (like `.Toastify__toast-container--top-left`)?

File: src/components/Toaster/component.tsx (hunk 1 above)
Comment (essj) [reply src/components/Toaster/component.tsx]: Done!

File: src/components/Toaster/styled.tsx (hunk 2 above)
Comment (essj) [reply src/components/Toaster/styled.tsx]: Yes, I've pushed a fix to order it better (without a newline).

**Review by cchungmin — APPROVED** [review]
Thanks! LGTM

**Pull request merged**